it/src/main/java/org/corfudb/universe/util/IpTablesUtil.java

```diff
@@ -0,0 +1,49 @@
+package org.corfudb.universe.util;
+
+/**
+ * Iptables wrapper/utility class
+ */
+public class IpTablesUtil {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)